timed/subscription/views.py

```diff
@@ -27,7 +28,16 @@ class SubscriptionProjectViewSet(viewsets.ReadOnlyModelViewSet):
     ordering_fields = ("name", "id")
 
     def get_queryset(self):
-        return Project.objects.filter(archived=False, customer_visible=True)
+        user = self.request.user
+        queryset = Project.objects.filter(archived=False, customer_visible=True)
+        if CustomerAssignee.objects.filter(user=user, is_customer=True).exists():
+            return queryset.filter(
+                Q(
+                    customer__customer_assignees__user=user,
+                    customer__customer_assignees__is_customer=True,
+                )
+            )
+        return queryset
```

Comment:
This returns all the projects if you're *NOT* a customer assignee. Not exactly sure this is what you want

Reply:
I suggest adding a check for user being employee before returning the full queryset